code/addons/docs/src/manager.tsx

```diff
@@ -14,6 +14,58 @@ import {
 import type { SourceParameters } from './blocks/blocks';
 import { Source } from './blocks/components/Source';
 
+const CodePanel = ({
+  active,
+  lastEvent,
+  currentStoryId,
+}: {
+  active: boolean | undefined;
+  lastEvent: any | undefined;
```

Comment:
I did try to type this, then I noticed that where it is emitted, `format` is not actually set....

https://github.com/storybookjs/storybook/blob/809c1304556822e991b1bca364755b2ba97ddd5d/code/core/src/preview-api/modules/preview-web/emitTransformCode.ts#L22-L26